sros2/sros2/policy/defaults/policy.xml

```diff
@@ -1,16 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<policy version="0.1.0">
-  <profiles>
-    <profile ns="/" node="default">
-      <topics publish="ALLOW" subscribe="ALLOW">
-        <topic>/*</topic>
-      </topics>
-      <services reply="ALLOW" request="ALLOW">
-        <service>/*</service>
-      </services>
-      <actions call="ALLOW" execute="ALLOW">
-        <action>/*</action>
-      </actions>
-    </profile>
-  </profiles>
+<policy version="0.1.0"
```

Comment:
this version here was the version of the policy file format?
should it be updated?